packages/app/src/app/pages/common/UserMenu/Menu/index.js

```diff
@@ -30,13 +37,23 @@ function Menu({ username, openPreferences, openStorageManagement, signOut }) {
           <SettingsIcon />
         </Icon>Preferences
       </Item>
+      {curator && (
+        <Link style={{ textDecoration: 'none' }} to={curatorUrl()}>
+          <Item>
+            <Icon>
+              <ImageIcon />
```

Comment:
For some reason I keep thinking of this icon when thinking of curators: ✨. Do you think it makes sense to use this icon instead? ✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨✨

Reply:
I love it!

Reply:
<img width="414" alt="screenshot 2018-11-23 at 19 10 27" src="https://user-images.githubusercontent.com/1051509/48956476-7cce0280-ef53-11e8-815a-4a0ed8fb2dd5.png">

Added :D